src/gui/src/components/explorer-grid/index.tsx

```diff
@@ -48,20 +49,23 @@ const getItemsData = (
   }
 
   const ids = new Set<string | undefined>(
-    allEdges.map(edge => edge.to?.id),
+    allEdges.map(edge =>
+      edge.to?.id ? baseDepID(edge.to.id) : undefined,
+    ),
   )
+  ids.delete(undefined)
   const stackable = ids.size > 1
   const sameItems = ids.size === 1
 
   for (const edge of allEdges) {
     if (sameItems && !edge.from && allEdges.length > 1) continue
-    const id = edge.to?.id
+    const _id = edge.to?.id
```

Comment:
```suggestion
    const id_ = edge.to?.id
```

Preceding underscore will make linting not catch if vars are unused at some future time.